pkg/sensors/tracing/generickprobe.go

```diff
@@ -897,11 +912,16 @@ func createKprobeSensorFromEntry(kprobeEntry *genericKprobe,
 	loadProgName, loadProgRetName := kernels.GenericKprobeObjs()
 	isSecurityFunc := strings.HasPrefix(kprobeEntry.funcName, "security_")
 
+	pinProg := kprobeEntry.funcName
+	if kprobeEntry.instance != 0 {
+		pinProg = fmt.Sprintf("%s_%d", kprobeEntry.funcName, kprobeEntry.instance)
```

Comment:
How about we use "%s:%d"? I'm just thinking that in theory there could be a fund name "foo_1" in the kernel that may cause collisions.

Reply:
ook, will change, thanks